Week-14/wealth-portfolio/src/middleware/errorMiddleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from "express";
+
+import httpStatus from "http-status";
+
+const errorHandler = (err: any, req: Request, res: Response, next: any) => {
+  let { statusCode, message } = err;
+  statusCode = httpStatus.INTERNAL_SERVER_ERROR;
```

Comment:
@apandyamob Are these default values for statusCode and message or you are overriding here ?

Reply:
In case something went wrong in the server exception will be caught and we will throw an internal server error.
We are not overriding default error status codes. @RajeshSivanesan